### PR TITLE
[OC-1439] New archives screen filters

### PR DESCRIPTION
**Pull request by vlo-rte**
Release notes : 

In Tasks section : 

[OC-1439](https://opfab.atlassian.net/browse/OC-1439) : New archives screen filters

### CODE REVIEW

File: ui/main/src/app/modules/archives/archives.component.html [hunk 1]
```diff
@@ -85,6 +127,9 @@
         <td translate [translateParams]="card.summary.parameters">
           {{ card.process + '.' + card.processVersion + '.' + card.summary.key}}
         </td>
+        <td translate>
+          {{ this.findServiceForProcess(card.process) }}
```
Comment (quinarygio) [inline ui/main/src/app/modules/archives/archives.component.html:131]: Check if displayServiceFilter() as for column header

**Review by quinarygio — COMMENTED** [review]
When selecting only the service and not the processes the query does not filters on selected "service"

File: ui/main/src/app/modules/archives/archives.component.html (hunk 1 above)
Comment (vlo-rte) [reply ui/main/src/app/modules/archives/archives.component.html]: Done.

**Comment by vlo-rte** [comment]
> When selecting only the service and not the processes the query does not filters on selected "service"

I've made an update to filter also on services (when no process selected). Thank you for your remark.

**Review by quinarygio — APPROVED** [review]

**Pull request merged**